StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory.swift

```diff
@@ -409,6 +424,22 @@ extension PaymentSheetFormFactory {
         label.numberOfLines = 0
         return StaticElement(view: label)
     }
+
+    func makeContactInformation(includeName: Bool, includeEmail: Bool, includePhone: Bool) -> SectionElement? {
+        let nameElement = includeName ? makeName() : nil
+        let emailElement = includeEmail ? makeEmail() : nil
+        let phoneElement = includePhone ? makePhone() : nil
+
+        let allElements: [Element?] = [nameElement, emailElement, phoneElement]
+        let elements = allElements.compactMap { $0 }
+
+        guard !allElements.isEmpty else { return nil }
```

Comment:
This should be checking `elements`, right?

Reply:
Good catch